### PR TITLE
New: use native slider instead of rangeslider (fixes #79)

### LINKED ISSUE #79: Use native slider instead of rangeslider
### Subject of the issue/enhancement/features

Would be good to try if we can use native input type=range for the Slider. [Browser support](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range) looks good.
Biggest issue will be a similar [cross browser design](https://css-tricks.com/styling-cross-browser-compatible-range-inputs-css/). This [demo](https://codepen.io/chriscoyier/pen/FtnAa) works and looks good in IE, Chrome and Firefox.

### Reasons

It seems that using a 3rd party library has the same time cost as the old self built slider, but with less control over the behaviour. We have now gotten rid of ie8>ie10 so it might be time to move back to using browser native behaviour.

See https://github.com/adaptlearning/adapt-contrib-slider/issues/161

### PR DESCRIPTION
**Pull request by chris-steele**
#79 

### New
* Use native slider instead of rangeslider

### Important
- Should be merged with https://github.com/adaptlearning/adapt-contrib-slider/pull/172

### CODE REVIEW

**Review by eleanor-heath — APPROVED** [review]

File: templates/confidenceSlider.jsx [hunk 1]
```diff
@@ -1,25 +1,82 @@
+import React, { useEffect, useRef, useState } from 'react';
 import Adapt from 'core/js/adapt';
-import React from 'react';
 import { classes, templates } from 'core/js/reactHelpers';
 
-export default function ConfidenceSlider (props) {
+export default function Slider (props) {
```
Comment (cahirodoherty-learningpool) [inline templates/confidenceSlider.jsx:5]: This should probably be kept as ConfidenceSlider

Comment (chris-steele) [reply templates/confidenceSlider.jsx]: Quite right; the perils of copy and paste!

**Comment by swashbuck** [comment]
The example.json could be improved to better align with adapt-contrib-slider's example.json.

`"_scaleEnd": 1,` could be changed to `"_scaleEnd": 10,` for a more realistic use case.

`"_showScaleNumbers": true,` is missing.

**Comment by swashbuck** [comment]
README.md - For the information at the bottom (ex. Version number, Framework versions, etc), could you use `<br>` tags for the line endings?  This will prevent line breaks from disappearing if the white space is removed.

line-endings

**Comment by swashbuck** [comment]
bower.json and package.json are missing "targetAttribute".

`"targetAttribute": "_confidenceSlider",`

**Comment by oliverfoster** [comment]
@chris-steele  bump

**Review by swashbuck — CHANGES_REQUESTED** [review]

**Review by swashbuck — APPROVED** [review]
👍

**Review by oliverfoster — APPROVED** [review]

**Pull request merged**